File: Java/백준/Bronze/20053. 최소， 최대 2/최소， 최대 2.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int t = in.nextInt();
        for (int i=0;i<t;i++){
            int a = in.nextInt();
            int max = -1000001;
            int min = 1000001;
            for (int j=0;j<a;j++){
                int n = in.nextInt();
                max = max>n?max:n;
                min = min<n?min:n;
            }
            System.out.printf("%d %d\n",min,max);
        }
    }
}

File: C#/백준/Bronze/20053. 최소， 최대 2/최소， 최대 2.cs
using System;
using System.Linq;

namespace q{
    class w{
        static void Main(String[] args){
            int t = int.Parse(Console.ReadLine());
            for (int i=0;i<t;i++){
                int a = int.Parse(Console.ReadLine());
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                Console.WriteLine($"{aa.Min()} {aa.Max()}");
            }
        }
    }
}